### ISSUE TITLE
use l1_norm.py

### ISSUE THREAD

**Issue by YJYTJ** [akshaybadola/decomposing-the-deep#1]
hi, could you please publish the code for training? I don't know how to use the l1_norm.py properly.
Thank you!

**Comment by akshaybadola** [comment]
Hi. apologies for the poor state of the code. Are you looking for the code for extracting the indices or for finetuning with the decomopsed final layer?

**Comment by YJYTJ** [comment]
Yes, I would like to ask about the usage of all the functions in l1_norm.py. I don't seem to see them appear during training. 
Thank you!

**Comment by akshaybadola** [comment]
Hi, again apologies for the delayed response. I've been quite busy last few days. I myself don't recall exactly how those were used, but we had conducted further experiments in another repo which is more current, but there are unstaged changes on my local disk and the README isn't updated. If you're willing to wait a couple of days I'll update that and I'll link that repo. It also has implementation of Selectivity Considered Harmful paper and comparison with that.

**Comment by YJYTJ** [comment]
Thank you so much! It's very kind of you.!

**Comment by YJYTJ** [comment]
Hi~ Is your code ready? I want to use yours as a comparison algorithm, and the deadline is approaching.

**Comment by akshaybadola** [comment]
Hi. I had gotten super busy again. But yes, I fixed the resnet20 part of the code. It's a lot cleaner. Please see repository https://github.com/akshaybadola/class_specific_cnn/. You can raise issues there as required.
